### ISSUE TITLE
Cannot set embedded language indentation rules

### ISSUE THREAD

**Issue by octref** [microsoft/vscode#73559]
Related: https://github.com/vuejs/vetur/issues/534

Vetur extension defines two languages: `vue` and `vue-html`.

I can only use `languages.setLanguageConfiguration` to set indentation rule of the outer language `vue` but not the embedded `vue-html`.

With this API call, here's what I see:

```ts
export function registerLanguageConfigurations() {
  // languages.setLanguageConfiguration('vue', {
  //   indentationRules: {
  // 		increaseIndentPattern: /<(?!\?|(?:area|base|br|col|frame|hr|html|img|input|link|meta|param)\b|[^>]*\/>)([-_\.A-Za-z0-9]+)(?=\s|>)\b[^>]*>(?!.*<\/\1>)|<!--(?!.*-->)|\{[^}"']*$/,
  // 		decreaseIndentPattern: /^\s*(<\/(?!html)[-_\.A-Za-z0-9]+\b[^>]*>|-->|\})/
  //   },
  // });
  languages.setLanguageConfiguration('vue-html', {
    indentationRules: {
      increaseIndentPattern: /<(?!\?|(?:area|base|br|col|frame|hr|html|img|input|link|meta|param)\b|[^>]*\/>)([-_\.A-Za-z0-9]+)(?=\s|>)\b[^>]*>(?!.*<\/\1>)|<!--(?!.*-->)|\{[^}"']*$/,
      decreaseIndentPattern: /^\s*(<\/(?!html)[-_\.A-Za-z0-9]+\b[^>]*>|-->|\})/
    }
  });
}
```

![bug](https://user-images.githubusercontent.com/4033249/57496844-e899f280-7289-11e9-94d0-7145034d56ca.gif)

If I remove the commented-out call to set `indentationRules` of `vue` language, then it seems to work. However, I'm worried it'll affect other non `vue-html` regions such as css/javascript.

You can repro by:

- Checking out this branch of Vetur: https://github.com/vuejs/vetur/tree/embedded-indentation-rules
- Compile with

  ```bash
  yarn
  cd server && yarn && cd ..
  yarn compile
  ```
- Run the `client` debug config to test it

In short:
- In `vue-html` embedded in `vue`, I would expect setting `vue-html`'s indentation rules would affect move line up/down
- I have to set both indentation rules for `vue` and `vue-html` to achieve this, which would have some side effect
